common/random_test.go

```diff
@@ -0,0 +1,108 @@
+package common_test
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/tendermint/tmlibs/common"
+)
+
+// It is essential that these tests run and never repeat their outputs
+// lest we've been pwned and the behavior of our randomness is controlled.
+// See Issues:
+//  * https://github.com/tendermint/tmlibs/issues/99
+//  * https://github.com/tendermint/tendermint/issues/973
+func TestUniqueRng(t *testing.T) {
+	if os.Getenv("TENDERMINT_INTEGRATION_TESTS") == "" {
+		t.Skipf("Can only be run as an integration test")
```

Comment:
why, too time consuming?

Reply:
In deed, yes it takes about 54s to run altogether.